chart/k8gb/crd/k8gb.absa.oss_gslbs.yaml

```diff
@@ -366,9 +372,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
looks like a fix for https://github.com/kubernetes-sigs/controller-tools/issues/456 , good to proceed 👍